app/src/fatec/pi/entities/User.java

```diff
@@ -0,0 +1,72 @@
+package fatec.pi.entities;
+
+public class User {
+	private Integer id;
+	private String name;
+	private String email;
+	private String password;
+	private Boolean admin;
```

Comment:
Não precisa, mas seria legal criar uma classe admin, que extende a classe usuário (herda todas funções/atributos).  Isso simplificaria bastante os checks de admin, e possibilitaria a criação de métodos especificos para administradores.

[isso deve ajudar](https://www.developer.com/java/data/implementing-generalization-and-specialization-in-java.html)